### PR TITLE
Add RICORD data creation and include in train/test split

### PR DESCRIPTION
**Pull request by andyzzhao**
# Pull Request Template

## Description

- Added create_ricord_dataset/create_ricord_dataset.ipynb to crop and convert RICORD data from DICOM to PNG.
- Added RICORD data processing to create_COVIDx.ipynb and create_COVIDx_binary.ipynb where 200 RICORD images are add to test in both notebooks

New workflow for creating COVIDx dataset:
1. Download required images and files
2. Run create_ricord_data/create_ricord_dataset.ipynb
3. Run create_COVIDx.ipynb and/or create_COVIDx_binary.ipynb

## Context of change

Please add options that are relevant and mark any boxes that apply.

- [x] Software (software that runs on the PC)
- [ ] Library (library that runs on the PC)
- [ ] Tool (tool that assists coding development)
- [ ] Other

## Type of change

Please mark any boxes that apply.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration.

create_COVIDx.ipynb and create_COVIDx_binary.ipynb were ran to confirm 200 RICORD images were added to test and the rest were added to train. 

## Checklist:

Please mark any boxes that have been completed.

- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by haydengunraj** [comment]
@andyzzhao this looks good, but we might hold off on merging until we're ready to release this along with the BIMCV data. We'll discuss at tomorrow's meeting.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,128 @@
+absl-py==0.10.0
```
Comment (haydengunraj) [inline requirements.txt:1]: Do we know if this is the minimal environment needed to use the repo? I'm happy to keep it as-is, but if there are any extraneous packages here I think we should remove them in a future update.

**Comment by haydengunraj** [comment]
@mayaliliya in the v8 label files, have we fixed the issues related to inconsistent labels that we had in the v7 dataset (e.g., #126)?

File: requirements.txt (hunk 1 above)
Comment (andyzzhao) [reply requirements.txt:1]: Yup it definitely can be cut down. I'll make another PR for it.

**Comment by mayaliliya** [comment]
> @mayaliliya in the v8 label files, have we fixed the issues related to inconsistent labels that we had in the v7 dataset (e.g., #126)?

I fixed it in the data.py script with the brute force pop method. I am thinking we just merge this and then I will do a separate PR to address this bug next week as well as thoroughly addressing the duplicate issue (i.e. seeing if we can scavenge more images rather than removing all images with the same url base).

File: docs/COVIDx.md [hunk 2]
```diff
@@ -22,38 +24,40 @@ The current COVIDx dataset is constructed by the following open source chest rad
...
  * go to this [link](https://www.kaggle.com/tawsifurrahman/covid19-radiography-database) to download the COVID-19 Radiography database. Only the COVID-19 image folder and metadata file is required. The overlaps between covid-chestxray-dataset are handled
  * go to this [link](https://www.kaggle.com/c/rsna-pneumonia-detection-challenge/data) to download the RSNA pneumonia dataset
+ * go to this [link] (https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=70230281) to download the RICORD COVID-19 dataset and clinical data csv
 2. Create a `data` directory and within the data directory, create a `train` and `test` directory
+3. Use [create\_ricord\_dataset\\create\_ricord\_dataset.ipynb](../create_ricord_dataset/create_ricord_dataset.ipynb) to pre-process the RICORD dataset before handling.
 3. Use [create\_COVIDx\_binary.ipynb](../create_COVIDx_binary.ipynb) to combine the three datasets to create COVIDx for binary classification. Make sure to remember to change the file paths. Use [create\_COVIDx.ipynb](../create_COVIDx.ipynb) for datasets compatible with COVIDx5 and earlier models (not binary classification).
```
Comment (haydengunraj) [inline docs/COVIDx.md:30]: This should be 4, and the next step is 5, It still renders correctly as-is though.

**Review by haydengunraj — APPROVED** [review]

**Pull request merged**